contextproject-ms3/src/main/java/contextproject/models/PlaylistProperty.java

```diff
@@ -0,0 +1,93 @@
+package contextproject.models;
+
+import javafx.beans.property.DoubleProperty;
+import javafx.beans.property.ObjectProperty;
+import javafx.beans.property.SimpleDoubleProperty;
+import javafx.beans.property.SimpleObjectProperty;
+import javafx.beans.property.SimpleStringProperty;
+import javafx.beans.property.StringProperty;
+
+public class PlaylistProperty {
```

Comment:
I think TrackProperty is more clear here. Because it's about all properties of a track row right?